Wrong shot number:
[16, 9, 2]

Correct shot number:
[1]

Raw json:

{'hasError': True, 'errParameter': ['dfs.namenode.max.objects'], 'reason': ["The property 'dfs.namenode.max.objects' has the value '1' which is not within the accepted value range {0, 1000000000}."]}

{'hasError': True, 'errParameter': ['dfs.namenode.max.objects'], 'reason': ["The property 'dfs.namenode.max.objects' has the value '1' which is not within the accepted value range {0, 2147483647}."]}

{'hasError': True, 'errParameter': ['dfs.namenode.max.objects'], 'reason': ["The property 'dfs.namenode.max.objects' has the value '1' which is not within the accepted value range {0, 2147483647}."]}

Final result:

There are 1 misconfiguration parameters in the input: dfs.namenode.max.objects
[Ciri] Reason for dfs.namenode.max.objects: The property 'dfs.namenode.max.objects' has the value '1' which is not within the accepted value range {0, 2147483647}.
